cms/static/js/views/pages/container.js

```diff
@@ -186,6 +186,21 @@ define(['jquery', 'underscore', 'backbone', 'gettext', 'js/views/pages/base_page
                     modal = new EditXBlockModal(options);
                 event.preventDefault();
 
+                  // WIP: now check if we want to launch in a modal for the new editors (behind waffle flag)
+                var useNewTextEditor = this.$('.edit-button').attr("use-new-editor-text"),
+                    useNewVideoEditor = this.$('.edit-button').attr("use-new-editor-video"),
+                    useNewProblemEditor = this.$('.edit-button').attr("use-new-editor-problem"),
+                    blockType = xblockElement.find('.xblock').attr("data-block-type");
+                console.log(useNewTextEditor,useNewVideoEditor,useNewProblemEditor, blockType);
```

Comment:
Is this for debugging purposes?

Reply:
Yeah I'll get rid of all these thanks for catching!